src/plugins/waterfall-plugin.ts

```diff
@@ -171,40 +181,46 @@ export default class WaterfallPlugin extends UIPlugin<WaterfallPluginStyles> {
                 this.renderEngine.options.tooltip(data, this.renderEngine, this.interactionsEngine.getGlobalMouse());
             } else {
                 const { data: index } = this.hoveredRegion;
-                const { name, intervals, timing, meta = [] } = this.data.find(({ index: i }) => index === i);
```

Comment:
find can also return a null, and then it would throw an exception in that case

Reply:
I think you can add `if` block for this case.)

Reply:
I added the if already in the changes, or you meant to do it diferently?

Reply:
Oh, sorry. Just did not notice the changes.) Everything is fine, don't worry.